Syntaxes/Julia.tmLanguage

```diff
@@ -244,13 +244,13 @@
 			<array>
 				<dict>
 					<key>match</key>
-					<string>\b(?:function|stagedfunction|type|immutable|macro|quote|abstract|bitstype|typealias|module|baremodule|new)\b</string>
```

Comment:
By a tiny percentage compare to the number of times it is used in a constructor.